internal/connect/api_test.go

```diff
@@ -250,3 +250,25 @@ func TestDeactivateProductError(t *testing.T) {
 		}
 	}
 }
+
+func TestProductMigrations(t *testing.T) {
+	createTestCredentials("", "", t)
+	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.Write(readTestFile("migrations.json", t))
+	}))
+	defer ts.Close()
+	CFG.BaseURL = ts.URL
+
+	migrations, err := productMigrations(nil)
+	if err != nil {
+		t.Fatal(err)
+	}
+	if len(migrations) != 2 {
+		t.Fatalf("len(migrations) == %d, expected 2", len(migrations))
+	}
+	newestBase := migrations[0][0].ToTriplet()
+	expected := "SLES/15.4/x86_64"
+	if newestBase != expected {
```

Comment:
this part of the test was added to check the sort worked, but `productMigrations()` doesn't sort anymore

Reply:
ok. I'll remove it.